modules/standard/BitOps.chpl

```diff
@@ -157,7 +157,7 @@ module BitOps {
  * module to hide the extern procedures
  */
 pragma "no doc"
```

Comment:
Seems like it may as well be changed now to avoid two single line changes. This module being marked private is correct though.